topics/contributing_to_the_framework.adoc

```diff
@@ -16,7 +16,7 @@ Currently, the best method for proposing a change to the _Community Collaboratio
 
 .Prerequisites
 
-You must be a member of the GitHub `community-collaboration-guide` repository.
+You must be a member of the GitHub `community-collaboration-guide` repository. Request to join by emailing your GitHub username to ccs-upstream@redhat.com.
```

Comment:
@pmkovar is this right?

Reply:
It is technically correct, yes, even though once we clarify the license in #119 we could start accepting drive-by contributions (think: typo fixes) without the need to become a member first.

Reply:
Got it! Will merge this for now :)